src/hooks/useWithdrawLiquidityHandler.tsx

```diff
@@ -34,13 +35,24 @@ export const useWithdrawLiquidityHandler = () => {
   }
 
   async function approveTransaction() {
-    // Deposit pool contract address
-    let address = process.env.NEXT_PUBLIC_DEPOSIT_POOL_ADDRESS
-    if (provider && address) {
-      if (accountInfo && accountInfo?.address) {
-        setdepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
+    // Position Manager contract address
+    let address =
+      process.env.NEXT_PUBLIC_ENVIRONMENT === 'local'
+        ? process.env.NEXT_PUBLIC_POSITION_MANAGER_ADDRESS || ''
+        : '0x3eFadc5E507bbdA54dDb4C290cc3058DA8163152'
+    if (provider) {
+      if (process.env.NEXT_PUBLIC_ENVIRONMENT === 'local') {
+        if (accountInfo && accountInfo?.address) {
```

Comment:
you can remove `accountInfo` since `accountInfo?.address` already checks for both

Reply:
Done